tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -1790,12 +1791,13 @@ class ParameterizedOpConverterTestBase
       TF_EXPECT_OK(
           TensorShapeUtils::MakeShape(expected_output_dims[i], &shape));
       string out_name = (n_output == 1) ? name : StrCat(name, ":", i);
+      DataType type = out_tf_type.size() > i ? out_tf_type[i] : tf_type;
```

Comment:
This is not a problem in your change, but I notice tf_type is missing a suffix _ as it is a class field. Also since we now distinguish input and output tf_type,  we may want to rename tf_type_ to in_tf_type. 
Can we also rename "type" to out_tf_type?